docs/sources/flow/reference/components/otelcol.exporter.otlp.md

```diff
@@ -107,6 +107,11 @@ server.
 
 {{< docs/shared lookup="flow/reference/components/otelcol-tls-config-block.md" source="agent" >}}
 
+> **NOTE**: `otelcol.exporter.otlp` uses gRPC, which disallows sending sensitive credentials (like `auth`) over insecure channels for security reasons.
+> If you need to send sensitive credentials over non-TLS connections (via the `insecure` argument), you may want to use [otelcol.exporter.otlphttp][] instead.
```

Comment:
```suggestion
> If you need to send sensitive credentials over non-TLS connections (via the `insecure` argument), you can use [otelcol.exporter.otlphttp][] instead.
```

Reply:
```suggestion
> Sending sensitive credentials over insecure non-TLS connections is supported by non-gRPC exporters such as [otelcol.exporter.otlphttp][.
```
What about this wording?

Reply:
From a language standpoint, that's good. It's just a clear statement without giving specific direction (which seems to be more appropriate here).

Reply:
I am fine with this, @rfratto WDYT?

Reply:
SGTM 👍